communication/contributor-comms/role-handbooks/Social-Media.md

```diff
@@ -13,28 +13,28 @@ Do note that tweets targeting the greater Kubernetes community should potentiall
 
 ## Minimum Skills and Requirements
 
-- Are a member of the [Kubernetes GitHub Org]
+- Are a member of the Kubernetes GitHub Org
 - Familiar with twitter by either having a personal account or managing another account (brand, oss project, employer, local nonprofit, etc)
-- Available to either attend regular Contributor Comms meetings or collaborate asynchronously via the #sig-contribex-comms channelnin the Kubernetes Slack
+- Available to either attend regular Contributor Comms meetings or collaborate asynchronously via the #sig-contribex-comms channel in the Kubernetes Slack
 
 ### Expected Time Investment
 
 Team leads: 3-5 hours per week.
-Team members: 1-3 hour per week.
+Team members: 1-3 hours per week.
 
 ## General Expectations
 
 - Rules for social posts and more can be found in [social guidelines](https://github.com/kubernetes/community/blob/master/communication/contributor-comms/storytelling-resources/social-guidelines.md)
 - Be welcoming, be yourself
-- Understanding, communicating, and setting/adjusting [social guidelines](https://github.com/kubernetes/community/blob/master/communication/contributor-comms/storytelling-resources/social-guidelines.md)
```

Comment:
This is not a typo. For more information: https://en.wikipedia.org/wiki/Serial_comma

Reply:
I have reverted this change too